srcpkgs/vifm/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'vifm'
 pkgname=vifm
-version=0.12
+version=0.13
 revision=1
 build_style=gnu-configure
 configure_args="--without-gtk"
 hostmakedepends="mdocml"
-makedepends="ncurses-devel file-devel"
+makedepends="ncurses-devel file-devel perl"
 depends="desktop-file-utils"
 short_desc="Ncurses-based file manager with vi-like keybindings"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="João Pinto <joaopintojpgp@gmail.com>"
```

Comment:
we'd prefer package get adopted by established contributors. I encourage you to keep track of this package and continue making contributions, but please don't change the maintainer at this time

https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#adopting-a-template

Reply:
Ok very well, I will have to see how I can get this to build this on musl and arm before commiting again.